tests/unittests/Foundation/NSPointerFunctionsTests.mm

```diff
@@ -80,7 +80,8 @@ NSUInteger NSPointerFunctionsTestStructSizeFunction(const void* item) {
     const void* ptr3 = (void*)((long)ptr1 + 1);
 
     testEqualFunction(functions, ptr1, ptr2, ptr3);
-    testHashFunction(functions, ptr1, ptr2, ptr3);
+    // Disabling hash function check here as performing hash on opaque values cannot be accuratly matched with the reference platform.
+    // testHashFunction(functions, ptr1, ptr2, ptr3);
```

Comment:
Discussed offline, we'll keep this as-is.

---
In reply to: [73766353](https://github.com/Microsoft/WinObjC/pull/833#discussion_r73766353) [](ancestors = 73766353,73765315,73764838,73764075,73762989)